src/audio-worklet/free-queue/src/interface/README.md

```diff
@@ -47,13 +53,11 @@ emcc \
 	-o {output_file_name.js}
 ```
 
-### Example Usage
-
 An example workflow of using interface.
 
-1. Create FreeQueue in C.
-2. Query for Pointers through JS.
-3. Create JS instance of FreeQueue through Pointers.
-4. Now you can push and pull from either side, while one side being producer and other being consumer.
+1. Create a FreeQueue in C.
```

Comment:
1. Create an instance of FreeQueue in the C code.
2. Get the pointer from the C FreeQueue instance.
3. Create a JS instance of FreeQueue with the obtained pointer.
4. Now push and pull from either side; one side being a producer and the other a consumer.